include/simdjson/implementation.h

```diff
@@ -10,6 +10,24 @@
 
 namespace simdjson {
 
+/**
+ * Validate the UTF-8 string.
+ *
+ * @param buf the string to validate.
+ * @param len the length of the string in bytes.
+ * @return true if the string is valid UTF-8.
+ */
+WARN_UNUSED bool validate_utf8(const char * buf, size_t length) noexcept;
+
+
+/**
+ * Validate the UTF-8 string.
+ *
+ * @param p the string_view to validate.
```

Comment:
p probably doesn't make sense here, assuming it means "padded string." I'd just use str or something. Nitpick.

Reply:
Point taken, fixed.

Reply:
Fixed.